docs/reference/settings/notification-settings.asciidoc

```diff
@@ -264,6 +264,13 @@ HTML feature groups>>.
 Set to `false` to completely disable HTML sanitation. Not recommended.
 Defaults to `true`.
 
+`xpack.notification.reporting.warning.kbn-csv-contains-formulas.text`::
+(<<dynamic-cluster-setting,Dynamic>>)
+Specifies a custom message to be sent if the formula verification criteria
+for CSV files, from {kib}'s {kib-ref}/reporting-settings-kb.html#reporting-csv-settings[`xpack.reporting.csv.checkForFormulas`], is true.
```

Comment:
I need some help here.  Could someone please check the Kibana link syntax for me?

I don't know what the correct syntax is for referencing Kibana documentation from within Elasticsearch documentation.  I guessed off another page I found, which was https://www.elastic.co/guide/en/elasticsearch/reference/7.16/how-monitoring-works.html.

Reply:
Resolved by @lockewritesdocs suggestion.